### PR TITLE
Fix alerts component/containers and CSS styling

### LINKED ISSUE #3266: Clean up alerts CSS
With feedback on https://github.com/LLK/scratch-gui/issues/2956

### PR DESCRIPTION
**Pull request by evhan55**
### Resolves

- Resolves #3266: Clean up alerts CSS
- Fixes React component/container model for alerts
- Progress towards #2956: Message System - Phase 1

### Still TODO

- Localization of alert messages
- Fix alerts api to be more decoupled/abstract/customizable

### CODE REVIEW

**Comment by evhan55** [comment]
@rschamp some notes:

- I think I reconfigured the alerts components/containers like you suggested now, but I didn't know how to use the spread operator on component props to just pass them forward instead of duplicating them in the containers
- I added some custom CSS to an existing component: `CloseButton`

**Comment by rschamp** [comment]
@evhan55 Since the expectations of some components changed, you just need to run `npm run test:unit -- -u` to update the Jest snapshots for unit tests to pass.

File: src/containers/alert.jsx [hunk 1]
```diff
@@ -15,10 +15,15 @@ class Alert extends React.Component {
...
+            message
+        } = this.props;
         return (
             <AlertComponent
-                iconURL={this.props.iconURL}
-                message={this.props.message}
+                iconURL={iconURL}
+                message={message}
                 onCloseAlert={this.handleOnCloseAlert}
             />
```
[the remark this answers is not in the document]
Comment (evhan55) [reply src/containers/alert.jsx]: For some reason, doing this causes the `this.handleOnCloseAlert` to not work properly, the binding is wrong on either `onCloseAlert` or `index`, I played around and nothing worked.  So, I didn't include the spread operator to pass props through this time around, maybe something to revisit later to see why it breaks the handler.

**Comment by evhan55** [comment]
> @evhan55 Since the expectations of some components changed, you just need to run `npm run test:unit -- -u` to update the Jest snapshots for unit tests to pass.

Cool, thanks!

**Pull request merged**